rotkehlchen/chain/gnosis/modules/omnibridge/decoder.py

```diff
@@ -0,0 +1,34 @@
+from typing import TYPE_CHECKING, Final
+
+from rotkehlchen.chain.evm.decoding.omnibridge.decoder import OmnibridgeCommonDecoder
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.types import ChainID
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.user_messages import MessagesAggregator
+
+BRIDGE_ADDRESS: Final = string_to_evm_address('0xf6A78083ca3e2a662D6dd1703c939c8aCE2e268d')
+TOKENS_BRIDGING_INITIATED: Final = b'Y\xa9\xa8\x02{\x9c\x87\xb9a\xe2T\x89\x98!\xc9\xa2v\xb5\xef\xc3]\x1ft\t\xeaO)\x14p\xf1b\x9a'  # noqa: E501
+TOKENS_BRIDGED: Final = b'\x9a\xfdG\x90~%\x02\x8c\xda\xca\x89\xd1\x93Q\x8c0+\xbb\x12\x86\x17\xd5\xa9\x92\xc5\xab\xd4X\x15Re\x93'  # noqa: E501
```

Comment:
It seems to me that this two topics are the same in both chains. No need to pass them to the class as an attribute. You can define them in the `evm/decoding/omnibridge/decoder.py` file